pyproject.toml

```diff
@@ -35,7 +36,7 @@ dependencies = [
     "scikit-learn >= 0.24",
     "numba >= 0.52",
     "xarray >= 0.16",
-    "verde >= 1.7",
+    "verde >= 1.8.1",
```

Comment:
Are you sure this is required here? It's an issue with Verde only in Python 3.12 and Verde 1.7 should still work with previous Pythons.

Reply:
I'm not sure what to do here. If you're running Python 3.12 you **need** Verde >= v1.8.1.

Reply:
Yeah, you're right. I guess we should just bump the requirement here. It doesn't really have a huge impact and Verde 1.8 didn't break anything.